src/core/handle.ml

```diff
@@ -186,7 +188,48 @@ let handle_cmd : sig_state -> p_command -> sig_state * proof_data option =
       out 3 "(symb) %s ≔ %a\n" x.elt pp_term t;
       let d = if op then None else Some(t) in
       let ss = Sig_state.add_symbol ss e Defin x a impl d in
-      (ss, None)
+      (ss, None, None)
+  | P_inductive(e, s, t, tl) ->
+      (* Obtaining of the global type *)
+      let type_symbol_head = match t with
+        | None -> { elt = P_Type ; pos = None }
+        | Some a -> a
+      in
+      (* Add the first symbol in the signature *)
+      let (ss, _, _) = handle_cmd ss
```

Comment:
Thanks to follow the style currently used. See CONTRIBUTING.md.

Reply:
I don't see the problem here. I'm not in this case:
```
| A(x) ->
>>>>let y = ... in
>>>>...
```

Reply:
I am talking of the following lines. As I won't be available the next 2 days, thanks to check this with @gabrielhdt .

Reply:
Regarding style, if a `let` binding is using several lines, the line is broken after the `=`,  so
```ocaml
let (ss,_,_) =
  handle_cmd ...
in
```

By the way, when defining records, use
```ocaml
t = { f  : t 
    (** Ocamldoc comment *)
    ; f2 : u
    (** Ocamldoc comment *) }
```
and we tend to put parentheses around arguments of constructors.